### ISSUE TITLE
Kim Builder fails when change VPN

### ISSUE THREAD

**Issue by eirisdg** [rancher-sandbox/rancher-desktop#1163]
### Rancher Desktop Version

0.7.1

### Rancher Desktop K8s Version

1.21.8

### What operating system are you using?

Windows

### Operating System / Build Version

Windows 11 21H1 / 19043.1110

### What CPU architecture are you using?

x64

### Windows User Only

Yes, OpenVPN.

### Actual Behavior

I installed Rancher Desktop when my company VPN was active.
But I dont need it allways actived, so in some momments I need to disconnect it.
When I'm not connected and try to run Rancher Desktop, Kim Builder is not in the same IP, so it tryes to run until it gives timeout.

Examples:
With VPN (active when installed):
```pp2021-12-27T08:30:19.706Z: Waiting for Kubernetes node IP to become 192.168.139.49...
2021-12-27T08:30:19.738Z: Keeping builder pod kube-image:builder-p268l - pod IP 192.168.139.49 in 192.168.139.49
2021-12-27T08:30:19.776Z: [object Object]
2021-12-27T08:30:19.798Z: Skipping kim reinstall: service is ready, and without --force
```

Without VPN:
```
2021-12-27T08:25:18.778Z: Waiting for Kubernetes node IP to become 192.168.139.49...
2021-12-27T08:25:18.830Z: Deleting stale builder pod kube-image:builder-bgldb - pod IP 172.21.139.88 not in 192.168.139.49
2021-12-27T08:25:18.857Z: Existing kim install invalid: incorrect endpoint address.
2021-12-27T08:25:18.857Z: Installing kim: kim builder install --force
2021-12-27T08:26:39.361Z: Failed to restart the kim builder: C:\Users\xxxx\AppData\Local\Programs\Rancher Desktop\resources\resources\win32\bin\kim.exe exited with code 1.
2021-12-27T08:26:39.361Z: Attempt to run 'kim install builder' => error: 'time="2021-12-27T09:25:19+01:00" level=info msg="Applying node-role `builder` to `pf2zvezt`"
time="2021-12-27T09:25:19+01:00" level=info msg="Asserting namespace `kube-image`"
time="2021-12-27T09:25:19+01:00" level=info msg="Asserting TLS secrets"
time="2021-12-27T09:25:20+01:00" level=info msg="Asserting service/endpoints"
time="2021-12-27T09:25:20+01:00" level=info msg="Installing builder daemon"
time="2021-12-27T09:25:20+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:25:25+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:25:32+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:25:38+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:25:43+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:25:49+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:25:55+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:26:01+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:26:06+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:26:11+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:26:17+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:26:23+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:26:28+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:26:34+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:26:39+01:00" level=info msg="Waiting on builder daemon availability..."
Error: timeout waiting for builder to become available
Usage:
  kim builder install [OPTIONS]

Flags:
      --agent-image string         Image to run the agent w/ missing tag inferred from version
      --agent-port int             Port that the agent will listen on (default 1233)
      --buildkit-image string      BuildKit image for running buildkitd (default "docker.io/moby/buildkit:v0.8.3")
      --buildkit-port int          BuildKit service port (default 1234)
      --buildkit-socket string     BuildKit socket address (default "unix:///run/buildkit/buildkitd.sock")
      --containerd-socket string   Containerd socket address (default on k3s "/run/k3s/containerd/containerd.sock")
      --containerd-volume string   Containerd storage volume (default on k3s "/var/lib/rancher")
      --force                      Force installation by deleting existing builder
  -h, --help                       help for install
      --no-fail                    Do not fail if backend components are already installed
      --no-wait                    Do not wait for backend to become available
      --selector string            Selector for nodes (label query) to apply builder role

Global Flags:
  -x, --context string      kubeconfig context for authentication
      --debug               
      --debug-level int     
  -k, --kubeconfig string   kubeconfig for authentication
  -n, --namespace string    namespace (default "kube-image")

time="2021-12-27T09:26:39+01:00" level=fatal msg="timeout waiting for builder to become available"
'
2021-12-27T08:26:39.361Z: A reset might be necessary to support building images.

```

My IPConfig WITH vpn active is:
```
Adaptador de Ethernet Ethernet 3:

   Estado de los medios. . . . . . . . . . . : medios desconectados
   Sufijo DNS específico para la conexión. . :

Adaptador desconocido Conexión de área local:

   Estado de los medios. . . . . . . . . . . : medios desconectados
   Sufijo DNS específico para la conexión. . :

Adaptador de Ethernet Ethernet:

   Estado de los medios. . . . . . . . . . . : medios desconectados
   Sufijo DNS específico para la conexión. . :

Adaptador desconocido OpenVPN Wintun:

   Estado de los medios. . . . . . . . . . . : medios desconectados
   Sufijo DNS específico para la conexión. . :

Adaptador de Ethernet Ethernet 5:

   Estado de los medios. . . . . . . . . . . : medios desconectados
   Sufijo DNS específico para la conexión. . : red.fnmt.rcm

Adaptador desconocido OpenVPN TAP-Windows6:

   Sufijo DNS específico para la conexión. . : f254e117-2a1c-4cf5-ba66-15441997aacc.privatelink.westeurope.azmk8s.io
   Vínculo: dirección IPv6 local. . . : fe80::34cb:a451:6414:802a%28
   Dirección IPv4. . . . . . . . . . . . . . : 172.27.241.11
   Máscara de subred . . . . . . . . . . . . : 255.255.248.0
   Puerta de enlace predeterminada . . . . . :

Adaptador de LAN inalámbrica Conexión de área local* 1:

   Estado de los medios. . . . . . . . . . . : medios desconectados
   Sufijo DNS específico para la conexión. . :

Adaptador de LAN inalámbrica Conexión de área local* 2:

   Estado de los medios. . . . . . . . . . . : medios desconectados
   Sufijo DNS específico para la conexión. . :

Adaptador de LAN inalámbrica Wi-Fi:

   Sufijo DNS específico para la conexión. . :
   Vínculo: dirección IPv6 local. . . : fe80::807c:7b81:ff10:f139%12
   Dirección IPv4. . . . . . . . . . . . . . : 192.168.0.16
   Máscara de subred . . . . . . . . . . . . : 255.255.255.0
   Puerta de enlace predeterminada . . . . . : 192.168.0.1

Adaptador de Ethernet Conexión de red Bluetooth 2:

   Estado de los medios. . . . . . . . . . . : medios desconectados
   Sufijo DNS específico para la conexión. . :

Adaptador de Ethernet vEthernet (WSL):

   Sufijo DNS específico para la conexión. . :
   Vínculo: dirección IPv6 local. . . : fe80::3903:ee32:8d6a:1414%87
   Dirección IPv4. . . . . . . . . . . . . . : 192.168.128.1
   Máscara de subred . . . . . . . . . . . . : 255.255.240.0
   Puerta de enlace predeterminada . . . . . :
```

### Steps to Reproduce

Install Rancher Deskton with VPN connected and try to use builder with VPN disconnected

### Result

```
2021-12-27T08:25:18.857Z: Installing kim: kim builder install --force
2021-12-27T08:26:39.361Z: Failed to restart the kim builder: C:\Users\adominguez\AppData\Local\Programs\Rancher Desktop\resources\resources\win32\bin\kim.exe exited with code 1.
2021-12-27T08:26:39.361Z: Attempt to run 'kim install builder' => error: 'time="2021-12-27T09:25:19+01:00" level=info msg="Applying node-role `builder` to `pf2zvezt`"
time="2021-12-27T09:25:19+01:00" level=info msg="Asserting namespace `kube-image`"
time="2021-12-27T09:25:19+01:00" level=info msg="Asserting TLS secrets"
time="2021-12-27T09:25:20+01:00" level=info msg="Asserting service/endpoints"
time="2021-12-27T09:25:20+01:00" level=info msg="Installing builder daemon"
time="2021-12-27T09:25:20+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:25:25+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:25:32+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:25:38+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:25:43+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:25:49+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:25:55+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:26:01+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:26:06+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:26:11+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:26:17+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:26:23+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:26:28+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:26:34+01:00" level=info msg="Waiting on builder daemon availability..."
time="2021-12-27T09:26:39+01:00" level=info msg="Waiting on builder daemon availability..."
Error: timeout waiting for builder to become available
Usage:
  kim builder install [OPTIONS]

Flags:
      --agent-image string         Image to run the agent w/ missing tag inferred from version
      --agent-port int             Port that the agent will listen on (default 1233)
      --buildkit-image string      BuildKit image for running buildkitd (default "docker.io/moby/buildkit:v0.8.3")
      --buildkit-port int          BuildKit service port (default 1234)
      --buildkit-socket string     BuildKit socket address (default "unix:///run/buildkit/buildkitd.sock")
      --containerd-socket string   Containerd socket address (default on k3s "/run/k3s/containerd/containerd.sock")
      --containerd-volume string   Containerd storage volume (default on k3s "/var/lib/rancher")
      --force                      Force installation by deleting existing builder
  -h, --help                       help for install
      --no-fail                    Do not fail if backend components are already installed
      --no-wait                    Do not wait for backend to become available
      --selector string            Selector for nodes (label query) to apply builder role

Global Flags:
  -x, --context string      kubeconfig context for authentication
      --debug               
      --debug-level int     
  -k, --kubeconfig string   kubeconfig for authentication
  -n, --namespace string    namespace (default "kube-image")

time="2021-12-27T09:26:39+01:00" level=fatal msg="timeout waiting for builder to become available"
'
2021-12-27T08:26:39.361Z: A reset might be necessary to support building images.

```

### Expected Behavior

It may run in any IP

**Comment by mcekovic** [comment]
I have the same issue.

**Comment by jandubois** [comment]
In RD 1.0.0-beta.1 kim has been removed and buildkitd runs as a regular service.

Please verify if you issue has been resolved by this, and open a new issue, if it hasn't.
